app/models/section.rb

```diff
@@ -20,4 +22,26 @@ def conflicts_with(section)
     end
     false
   end
+  
+  def conflicts
+    Redis.current.smembers id
+  end
+  
+  private
+  def type_cast_periods_start_end
```

Comment:
this method should not be needed, please remove. it was a bad workaround to a bug.